pincer/middleware/interaction_create.py

```diff
@@ -3,8 +3,8 @@
 
 from __future__ import annotations
 
-from contextlib import suppress
 import logging
+from contextlib import suppress
 from inspect import isasyncgenfunction, _empty
 from typing import Dict, Any
 from typing import TYPE_CHECKING
```

Comment:
Shouldn't logging be moved under inspect so its alphabetical?

Reply:
I have always seperated `import` and `from` when sorting my imports

Reply:
yeah, that seems reasonable.